pls_cli/please.py

```diff
@@ -131,6 +132,45 @@ def quotes(show: bool = True) -> None:
     )
 
 
+class language(str, Enum):
+    english = "english"
+    spanish = "spanish"
+
+
+@app.command("language", rich_help_panel="Utils and Configs")
+def language(
+    lang: language = typer.Option(language.english, case_sensitive=False)
+) -> None:
+    """Choose language 🛩️"""
+    settings = Settings().get_settings()
+    if settings["language"] == lang:
```

Comment:
here it would be better to have just an if with the comparison of different. And in the end have only one center_print to avoid code duplication, like this:
```py
if settings["language"] != lang:
    settings["language"] = lang
    Settings().write_settings(settings)
    # Here function Dowload quotes
    center_print(
        Rule(
            "Thanks for letting me know that!",
            style=insert_or_delete_line_style,
        ),
        style=insert_or_delete_text_style,
    )

cener_print(
    Rule(
        f"Current language is: {lang}",
        style=insert_or_delete_line_style,
    ),
    style=insert_or_delete_text_style,
)

```